### PR TITLE
add the name of the bd to the user_menu when debug mode is active

### PR DESCRIPTION
**Pull request by FrancoisGe**

### CODE REVIEW

File: addons/wowl/static/src/components/user_menu/user_menu.ts [hunk 1]
```diff
@@ -27,12 +27,16 @@ export class UserMenu extends Component<{}, OdooEnv> {
...
   source: string;
   user = useService("user");
+  displayName = this.user.name;
 
   constructor() {
     super(...arguments);
     const { origin } = this.env.browser.location;
     const { userId } = this.user;
     this.source = `${origin}/web/image?model=res.users&field=image_128&id=${userId}`;
+    if (this.env.debug) {
```
Comment (kebeclibre) [inline addons/wowl/static/src/components/user_menu/user_menu.ts]: I would have put this in the template
`<t t-if="env.debug" t-esc="dbName" />`

Comment (FrancoisGe) [reply addons/wowl/static/src/components/user_menu/user_menu.ts]: Like in the last commit ?

File: addons/wowl/static/src/types.ts [hunk 2]
```diff
@@ -65,6 +65,7 @@ export interface SessionInfo {
 
 export interface Odoo {
   session_info: SessionInfo;
+  debug: string;
```
Comment (ged-odoo) [inline addons/wowl/static/src/types.ts:71]: any reason why it is a string and not a boolean? I don't think we want to make a difference between assets and just debug

Comment (FrancoisGe) [reply addons/wowl/static/src/types.ts:71]: In legacy, it sometimes differentiates the assets mode from the default debug mode.
Maybe we could use  'isDebug(type?:string)'  which returns true if we are in debug mode of the type in param ?

**Review by ged-odoo — APPROVED** [review]

**Pull request merged**